ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -356,6 +356,8 @@ literals:
   - OFF
   - AUTO
   - MANUAL
+  - STANDBY
+  - WORKING
```

Comment:
ON?

Reply:
The run mode only has a STANDBY/WORKING states to indicate which mode the unit is currently operating. ON is provided separately as a run_status.

Reply:
but what is the difference between `WORKING` and `ON`? I dont want the list of possible allowed states to get unnecessarily large, especially if they mean the same thing in this case. What is the difference?

Reply:
There are multiple units. They are designated as working or standby units. Working units may be ON or OFF depending on the parameters being controlled.